spec/form_objects/support_ticket/describe_yourself_form_spec.rb

```diff
@@ -8,16 +8,19 @@
   end
 
   describe '#descibe_yourself_options' do
-    it 'returns an array of label & value pairs' do
-      stub_const('SupportTicket::DescribeYourselfForm::OPTIONS', { type_of_user_value: 'Type of user label' })
-      expect(form.describe_yourself_options).to match_array([OpenStruct.new(value: :type_of_user_value, label: 'Type of user label')])
+    it 'returns an array of SupportTicket::Options' do
+      stub_const('SupportTicket::DescribeYourselfForm::OPTIONS', [{ value: :type_of_user_value, label: 'Type of user label' }])
+      expect(form.describe_yourself_options).to be_an(Array)
+      expect(form.describe_yourself_options.first).to be_a(SupportTicket::Option)
+      expect(form.describe_yourself_options.first.value).to eq(:type_of_user_value)
+      expect(form.describe_yourself_options.first.label).to eq('Type of user label')
     end
   end
 
   describe '#selected_option_label' do
     it 'returns label for the selected option' do
-      stub_const('SupportTicket::DescribeYourselfForm::OPTIONS', { option_1: 'Hello world' })
-      expect(form.selected_option_label('option_1')).to eq('Hello world')
+      stub_const('SupportTicket::DescribeYourselfForm::OPTIONS', [{ value: :option_1, label: 'Hello world' }])
+      expect(form.selected_option_label(:option_1)).to eq('Hello world')
```

Comment:
I really like the proper use of symbols to identify something than using strings! Lovely